apps/angular/src/core/interceptors/header.interceptor.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpEvent, HttpHandlerFn, HttpRequest } from '@angular/common/http';
+import { inject } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { ApiConfig } from '../services/api-config.service';
+
+/**
+	* Interceptor add header.
+	* @param req - Outgoing HttpRequest.
+	* @param next - Function representing the next processing step in the interceptor chain.
```

Comment:
Please omit `-` after params in jsdoc
```suggestion
	* @param req Outgoing HttpRequest.
	* @param next Function representing the next processing step in the interceptor chain.
```